src/index.js

```diff
@@ -40,8 +34,10 @@ export async function run() {
       await execShellCommand('pacman -Sy --noconfirm tmate');
       tmateExecutable = 'CHERE_INVOKING=1 tmate'
     } else {
-      if (core.getInput('install_dependencies') === "true") {
-        await installDependenciesLinux(optionalSudoPrefix);
+      if (core.getInput("install-dependencies") === "true") {
+        const optionalSudoPrefix = core.getInput("sudo") === "true" ? "sudo " : "";
+        await execShellCommand(optionalSudoPrefix + "apt-get update");
+        await execShellCommand(optionalSudoPrefix + "apt-get install -y openssh-client xz-utils");
       }
```

Comment:
Hmm. What about macOS and Windows? If anybody wants to use this Action on their self-hosted runner, they might want to skip installing the dependencies there, too.

I would have expected the `if(...install-dependencies...)` to span even the line `if (process.platform === "darwin") {` above.

Reply:
I've got your point. I agree with you!
I will change codes and push.